### PR TITLE
Update help text

### PR DESCRIPTION
**Pull request by emiloberg**

### CODE REVIEW

File: src/client/Modal.elm [hunk 1]
```diff
@@ -109,7 +109,7 @@ Expert rules are for advanced path matching and bulk operations and should be us
 
 ##### Matches using Regex
 
-Expert rules use a javascript regex syntax to match incoming rules
+Expert rules use a javascript regex syntax to match incoming rules. Start and end of string anchors (^ and $) are automatically added to the regex and must not be added manually.
```
Comment (marreman) [inline src/client/Modal.elm]: How difficult would it be to enforce this programmatically?

**Comment by emiloberg** [comment]
Simple regex in the schema, we should do it. Created this https://izettle.atlassian.net/browse/WEB-245

**Comment by marreman** [comment]
Lovely!

**Comment by emiloberg** [comment]
Thank you!

**Pull request merged**